README.md

```diff
@@ -40,13 +40,13 @@ Full documentation for deploying Bitwarden with Docker can be found in our help
 - [Docker](https://www.docker.com/community-edition#/download)
 - [Docker Compose](https://docs.docker.com/compose/install/) (already included with some Docker installations)
 
-*These dependencies are free to use.*
+_These dependencies are free to use._
 
 ### Linux & macOS
 
 ```
-curl -s -o bitwarden.sh \
-    https://raw.githubusercontent.com/bitwarden/server/master/scripts/bitwarden.sh \
+curl -s -L -o bitwarden.sh \
+    "https://func.bitwarden.com/api/dl/?app=self-host&platform=linux" \
```

Comment:
ℹ️ I didn't personally know about this subdomain, but whoever owns it might want to set up some defaults as https://func.bitwarden.com loads an Azure Function App banner / page.